sql/item_mods.sql

```diff
@@ -25407,10 +25500,10 @@ INSERT INTO `item_mods` VALUES (21374,302,1);
 INSERT INTO `item_mods` VALUES (21376,1,10);
 INSERT INTO `item_mods` VALUES (21376,25,10);
 INSERT INTO `item_mods` VALUES (21377,541,4);
-INSERT INTO `item_mods` VALUES (21378,165,1); -- Yetshila - Critical hit rate +1%
```

Comment:
being the one that added Yetshila recently, the item description form ffxiah actually  says "Critical hit rate +1% Critical hit damage +5%" and "Critical hit rate +2% Critical hit damage +6%" which is why i didn't shorten it ;)